src/main/java/io/stargate/sgv2/jsonapi/api/exception/GenericExceptionMapper.java

```diff
@@ -11,8 +12,9 @@
  */
 public class GenericExceptionMapper {
 
-  @ServerExceptionMapper()
-  public RestResponse<CommandResult> genericExceptionMapper(Exception e) {
+  // explicitly add types to override Quarkus mappers
+  @ServerExceptionMapper({Exception.class, MismatchedInputException.class})
```

Comment:
Ok more specific matching makes sense; what I wasn't sure was if Quarkus had such more specific mappers.
Since it sounds like it does -- then, yes, we need to do this.

One caveat is that we may need to add similar overrides for other Jackson exception types then as well: we can add them as needed.